src/test/java/test/integration/cube/CubeCreateTestCommandTest.java

```diff
@@ -40,53 +39,50 @@ public void should_create_test_for_kubernetes() throws TimeoutException, FileNot
     @Test
     public void should_create_test_for_docker() throws TimeoutException, FileNotFoundException {
 
-        shell().execute("mkdir src/test/resources").execute("touch src/test/resources/Dockerfile");
-
         shell().execute("arquillian-setup --standalone --test-framework junit")
+            .execute("touch src/test/resources/Dockerfile")
             .execute("arquillian-cube-setup --type docker --file-path src/test/resources/Dockerfile");
 
         shell().execute("arquillian-create-test --named MyDockerTest --target-package org.cube.docker")
-            .execute("arquillian-cube-create-test --test-class org.cube.docker.MyDockerTest");
+            .execute("arquillian-cube-create-test --test-class org.cube.docker.MyDockerTest --exposed-port 8080 --container-name hello-world");
 
         final JavaClassSource testClass = extractClass(project, "org.cube.docker.MyDockerTest");
 
         assertThat(testClass).hasAnnotation(RunWith.class).withValue("org.jboss.arquillian.junit.Arquillian");
 
         assertThat(testClass).hasMethod("dockerUrlShouldNotBeNull");
         assertThat(testClass).hasField("url").annotatedWith(ArquillianResource.class).ofType("java.net.URL");
-        assertThat(testClass).hasField("url").annotatedWith("DockerUrl");
+        assertThat(testClass).hasField("url").hasAnnotation("DockerUrl").withPropertyAndValues("containerName:hello-world", "exposedPort:8080");
     }
 
     @Test
     public void should_create_test_for_docker_compose() throws TimeoutException, FileNotFoundException {
 
-        shell().execute("touch docker-compose.yml");
-
         shell().execute("arquillian-setup --standalone --test-framework junit")
+            .execute("touch docker-compose.yml")
             .execute("arquillian-cube-setup --type docker-compose --file-path docker-compose.yml");
 
         shell().execute("arquillian-create-test --named MyDockerComposeTest --target-package org.cube.docker")
-            .execute("arquillian-cube-create-test --test-class org.cube.docker.MyDockerComposeTest");
+            .execute("arquillian-cube-create-test --test-class org.cube.docker.MyDockerComposeTest  --exposed-port 8080 --container-name hello-world");
 
         final JavaClassSource testClass = extractClass(project, "org.cube.docker.MyDockerComposeTest");
 
         assertThat(testClass).hasAnnotation(RunWith.class).withValue("org.jboss.arquillian.junit.Arquillian");
 
         assertThat(testClass).hasMethod("dockerUrlShouldNotBeNull");
         assertThat(testClass).hasField("url").annotatedWith(ArquillianResource.class).ofType("java.net.URL");
-        assertThat(testClass).hasField("url").annotatedWith("DockerUrl");
+        assertThat(testClass).hasField("url").hasAnnotation("DockerUrl").withPropertyAndValues("containerName:hello-world", "exposedPort:8080");
```

Comment:
yeah but instead of repeating it multiple times with same name, I have used it in plural form